### PR TITLE
adds: ELK stack initial configuration and Telescope Integration

### PR DESCRIPTION
**Pull request by raygervais**
## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [X] **New Feature**: Change which adds functionality
- [X] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This adds the initial integration of Telescope's logging of posts to Kibana, which can be accessed via kibana.docker.localhost. 
image
image

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [X] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by raygervais** [comment]
Note, it appears Telescope backend / frontend aren't talking to each other. Let me see if I can fix that here prior to merge.

File: env.example [hunk 1]
```diff
@@ -137,3 +137,8 @@ GITHUB_TOKEN=
 
 # If we wish to override default collection
 # UNSPLASH_COLLECTION_ID=""
+
+
+
+# Microservice Development
```
Comment (humphd) [inline env.example]: Needed in staging/prod envs too.

File: src/api/docker-compose.yaml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 version: '3'
 
 services:
+  # Infrastructure
   reverse-proxy:
     image: traefik:v2.4
     command: --api.insecure=true --providers.docker
```
Comment (humphd) [inline src/api/docker-compose.yaml]: Remove this

**Review by humphd — CHANGES_REQUESTED** [review]
A couple small things, then this is good to land.  Can you also rebase on my latest changes?

File: src/api/docker-compose.yaml [hunk 3]
```diff
@@ -16,14 +16,67 @@ services:
       - '8080:8080'
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - ./traefik/config/traefik.yaml:/etc/traefik/traefik.yaml"
```
Comment (humphd) [inline src/api/docker-compose.yaml]: Stray `"` at the end?

Also, are we using this file (we don't have one in the tree)?  Should we add one or do it all through labels?

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by raygervais** [comment]
@humphd, did you want me to add Telescope in it's half-broken state here as well? Originally I had planned on it, but may be best to do that as a separate iteration.

**Review by humphd — APPROVED** [review]
Looks good, can I merge?

**Comment by raygervais** [comment]
Yup, this PR doesn't include the Telescope docker container anymore and moreless adds the ELK stack.

**Comment by humphd** [comment]
OK, let's figure out how to connect with telescope elsewhere.  I'm not sure the right move yet, and it shouldn't block this work.

**Pull request merged**